pages/pipelines/branch_configuration.md.erb

```diff
@@ -58,12 +56,12 @@ When you define a value in the `branch` attribute of a step, only builds on the
 
```

Comment:
When combining positive and negative patterns, any positive pattern must match, and every negative pattern must not match.

Reply:
I first wrote this as include/exclude, but we're def using positive/ngative?